spec/views/live/show_form.html.erb_spec.rb

```diff
@@ -116,7 +116,13 @@
     end
   end
 
-  xit "contains a link to edit the form" do
-    expect(rendered).to have_link("Edit this form", href: form_path(form.id, edit: true))
+  it "does not contain a link to create a new draft" do
+    expect(rendered).not_to have_link(t("show_live_form.draft_create"), href: form_path(form.id))
+  end
```

Comment:
I'm not sure if it's necessary to have the negative test? We could explicitly turn the feature flag off for this test so that it doesn't fail when the flag switches

Reply:
I did think about explicitly  turning off the feature flag but that wouldn't have caught the scenario where someone accidentally removes the feature flag clause on the page which then renders the button. regardless whether the flag is removed or turned off we shouldn't be expecting to see the button.